lib/auth/auth_with_roles.go

```diff
@@ -1822,9 +1832,12 @@ func (r resourceChecker) CanAccess(resource types.Resource) error {
 		}
 	case types.SAMLIdPServiceProvider:
 		return r.CheckAccess(rr, state)
+
+	case services.UnifiedResource153Adapter[services.IdentityCenterAccount]:
+		return r.CheckAccess(rr, state)
```

Comment:
Is there any UT that covers this flow if not can we add test coverage for the IdentityCenterAccount CheckAccess flow.

Reply:
Added in e055c04